extra-wm/i3status-rust/autobuild/defines

```diff
@@ -0,0 +1,5 @@
+PKGNAME=i3status-rust
+PKGSEC=x11
+PKGDEP="pulseaudio"
+BUILDDEP="rustc"
+PKGDES="Very resourcefriendly and feature-rich replacement for i3status, written in pure Rust"
```

Comment:
Please use descriptive and neutral remarks in `PKGDES=`, here's an acceptable example:

```
PKGDES="A Rust re-implementation of i3status"
```